wiki/Skinning/Tutorial/en.md

```diff
@@ -1,105 +1,136 @@
+
+Every sprite has an HD counterpart, even all frames in an animation can have HD counterparts. As a result, the filesize of the folder or archive will increase due to having more images than normal.
+
+All HD sprites may also be bigger in filesize due to the fact that the canvas size used is four times bigger compared to the normal sprite. There are essentially two resolution modes _osu!_ is using. Each of them prefers one set of sprites. The first mode is **LowResolution** while the second mode is **HighResolution.**
+
+- *LowResolution mode* uses the normal sprites and ignores the HD sprites **(SD-resolution skin)**
+- *HighResolution mode* prefers HD sprites and uses normal sprites as a fallback if no HD sprite is available **(HD-resolution skin)**
+
 ### Can someone make this skin from that show/game?
 
-If you have gone through the entire osu!skinning forums and you are certain that you cannot find _that_ skin, then congrats, osu! doesn't have it yet.
-With this in mind, you can take this initiative to create the skin you dream of and make it something that everyone else may want! 
+If you have gone through the entire osu!skinning forums and you are certain that you cannot find _that_ skin, then congrats, osu! doesn't have it yet. With this in mind, you can take this initiative to create the skin you dream of and make it something that everyone else may want! 
 
-Nonetheless, please, **never** request for a skin anywhere in the osu!fourms.
-Failing to follow this rule will your thread be moved to the wastelands.
+Nonetheless, please, **never** request for a skin anywhere in the osu!forums. Failing to follow this rule will your thread be moved to the wastelands.
 
 ### Where can I get this skin that a YouTuber/streamer played with?
 
-First off, don't go asking this in the osu!forums, your thread will most likely be thrown into the wastelands.
-Secondly, your best bet is to simply ask that person for the skin they're using.
+First off, don't go asking this in the osu!forums, your thread will most likely be thrown into the wastelands. Secondly, your best bet is to simply ask that person for the skin they're using.
 
 ### I want to make a skin! But what do I need?
 
-Well nice choice, but remember, skinning can be little hard for a newbie.
-In the beginning of making any skin, you should have:
+Well nice choice, but remember, skinning can be little hard for a newbie. In the beginning of making any skin, you should have:
 
-- beginner skills with any photo editing program (that supports transparency) like Photoshop, GIMP, Paint.NET, etc.
-- patience (yes, skinning does take some time)
-- an idea! (yes, this is **very** important. Try be clever when making your skin)
+- Beginner skills with any photo editing program (that supports transparency) like Photoshop, GIMP, Paint.NET, etc.
+- Patience (yes, skinning does take some time)
+- An idea! (yes, this is **very** important. Try be clever when making your skin!)
```

Comment:
an idea -> a good idea

try be -> try to be

Reply:
I'm going to leave `an idea` as is, not all ideas have to be good to be a good skin 🤷‍♂️